packages/server/lib/browsers/firefox.ts

```diff
@@ -406,36 +387,27 @@ async function recordVideo (videoApi: RunModeVideoApi) {
 }
 
 export async function open (browser: Browser, url: string, options: BrowserLaunchOpts, automation: Automation): Promise<BrowserInstance> {
-  // see revision comment here https://wiki.mozilla.org/index.php?title=WebDriver/RemoteProtocol&oldid=1234946
-  const hasCdp = browser.majorVersion >= 86
   const defaultLaunchOptions = utils.getDefaultLaunchOptions({
     extensions: [] as string[],
     preferences: _.extend({}, defaultPreferences),
     args: [
-      '-marionette',
       '-new-instance',
-      '-foreground',
       // if testing against older versions of Firefox to determine when a regression may have been introduced, uncomment the '-allow-downgrade' flag.
       // '-allow-downgrade',
       '-start-debugger-server', // uses the port+host defined in devtools.debugger.remote
       '-no-remote', // @see https://github.com/cypress-io/cypress/issues/6380
     ],
   })
 
-  let remotePort
-
-  if (hasCdp) {
-    remotePort = await getRemoteDebuggingPort()
-
-    defaultLaunchOptions.args.push(`--remote-debugging-port=${remotePort}`)
-  }
-
   if (browser.isHeadless) {
     defaultLaunchOptions.args.push('-headless')
     // we don't need to specify width/height since MOZ_HEADLESS_ env vars will be set
     // and the browser will spawn maximized. The user may still supply these args to override
-    // defaultLaunchOptions.args.push('--width=1920')
-    // defaultLaunchOptions.args.push('--height=1081')
+    // defaultLaunchOptions.args.push('-width=1920')
+    // defaultLaunchOptions.args.push('-height=1081')
+  } else if (os.platform() === 'win32' || os.platform() === 'darwin') {
+    // lets the browser come into focus. Only works on Windows or Mac
```

Comment:
Maybe a small blurb on what happens in `linux` if you pass it so if someone in the future tests is out they know what to look for. Does firefox still come to the foreground? If so, maybe mention that as well.

Reply:
added in 66b5de67fe